sdk/spring/azure-spring-cloud-stream-binder-eventhubs/README.md

```diff
@@ -38,10 +38,12 @@ that's why you have to fill `spring.cloud.stream.eventhub.checkpoint-storage-acc
 
 #### Partitioning Support
 
-Event Hub provides similar concept of physical partition as Kafka. But unlike Kafka's auto rebalancing between 
-consumers and partitions, Event Hub provides a kind of preemptive mode. Storage account acts as lease to 
-determine which partition is owned by which consumer. When a new consumer starts, it will try to steal some partitions 
-from most heavy-loaded consumer to achieve workload balancing.
+Event Hub provides a similar concept of physical partition as Kafka. But unlike Kafka's auto rebalancing between consumers and partitions, Event Hub provides a kind of preemptive mode. The storage account acts as a lease to determine which partition is owned by which consumer. When a new consumer starts, it will try to steal some partitions
+from most heavy-loaded consumers to achieve the workload balancing.
+
+The default partition key uses the hash code value of byte[] type `Message.payload`.
+You can use the configuration of Producer `partition-key-extractor-name`, or `partition-key-expression` to specify
+how to obtain the partition key to calculate the partition ID.
```

Comment:
The last paragraph talks about the partition concept of Event Hubs, but this paragraph talks about Spring Cloud Stream implementation. So perhaps we could delete this directly.